### DISCUSSION TITLE
How to link a milestone to a Project?

### DISCUSSION THREAD

**Discussion by yyang5823** [community/community#77727] [New to GitHub]
Is it possible to add one or more milestones to a an exsiting project?

**Comment by leeringrose** [comment]
Yes. You can add more than one milestone to a project.

**Reply by yyang5823** [reply]
So far, I have not been able to find how to do it?  Could you let me know the steps of adding a milestone to a project?  Certain features are not available to private repository with a free account.  Is this one of those?

**Comment by ignacio-borrajo** [comment]
No, you cannot add a milestone to a project, you can add a milestone to an issue.
I wish to be able to do so buy right now you must create a milestone in a repository and associate issue to this milestone. I will open an issue to be able to create the milestone in a project and associate any issue to a project milestone.
